awx/ui_next/src/screens/Template/WorkflowJobTemplateEdit/WorkflowJobTemplateEdit.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { CardBody } from '@components/Card';
+import { getAddedAndRemoved } from '@util/lists';
+import { WorkflowJobTemplatesAPI, OrganizationsAPI } from '@api';
+import ContentLoading from '@components/ContentLoading';
+import { WorkflowJobTemplateForm } from '../shared';
+
+function WorkflowJobTemplateEdit({ template, hasContentLoading }) {
+  const [formSubmitError, setFormSubmitError] = useState();
+  const history = useHistory();
+
+  const handleSubmit = async values => {
+    const { labels, ...remainingValues } = values;
+    try {
+      await WorkflowJobTemplatesAPI.update(template.id, remainingValues);
+      await Promise.all([submitLabels(labels, values.organization)]);
+      history.push(`/templates/workflow_job_template/${template.id}/details`);
+    } catch (err) {
+      setFormSubmitError(err);
+    }
+  };
+  const submitLabels = async (labels = [], orgId) => {
+    const { added, removed } = getAddedAndRemoved(
+      template.summary_fields.labels.results,
+      labels
+    );
+    if (!orgId && !template.organization) {
```

Comment:
spoke to @jlmitch5  about this use case, which is if no Org is selected in the form and there is no org associated with the template and I generate a label (which needs an org ID to post to api) which org do I use.  Looks like old ui makes a call to the api, gets list of orgs and picks the first one in the list. Seems weird to me so I definitely open to suggestions about this.